src/Build/Resources/Strings.resx

```diff
@@ -478,6 +478,10 @@
     <value>MSB4110: Expected a property at position {1} in condition "{0}". Did you forget the opening parenthesis after the '$'? To use a literal '$', use '%24' instead.</value>
     <comment>{StrBegin="MSB4110: "}</comment>
   </data>
+  <data name="IllFormedPropertyWhitespaceInCondition" xml:space="preserve">
+    <value>MSB4151: Unexpected whitespace at position "{1}" of condition "{0}". Did you forget to remove a whitespace?</value>
+    <comment>{StrBegin="MSB4151: "}</comment>
```

Comment:
```suggestion
    <value>MSB4259: Unexpected whitespace at position "{1}" of condition "{0}". Did you forget to remove a whitespace?</value>
    <comment>{StrBegin="MSB4259: "}</comment>
```

Reply:
(4151 was deprecated, but we should still avoid reusing error codes so people can search for them effectively online.)

Reply:
Ok